meta-cube/recipes-support/overc-utils/source/cube-cfg

```diff
@@ -803,8 +803,10 @@ case "${cmd}" in
 	clean_config_artifacts
 	;;
     show)
+	container_name=${cmd_options_non_dashed[1]}
```

Comment:
-n is globally valid.

So it has to be checked as providing a name before throwing an error.